app/routes/pokedex/components/Yae.tsx

```diff
@@ -1,17 +1,39 @@
 import { useState, useEffect } from "react";
+import * as v from "valibot";
 import "../styles/trying.css";
 
+const PokemonSchema = v.object({
+  id: v.pipe(v.number()),
+  name: v.pipe(v.string(), v.minLength(1)),
+  types: v.array(v.string()),
+  img: v.pipe(v.string(), v.minLength(1)),
+  url: v.pipe(v.string(), v.minLength(1)),
+});
+type Pokemons = v.InferOutput<typeof PokemonSchema>;
+
+const TypesSchema = v.object({
+  name: v.pipe(v.string(), v.minLength(1)),
+});
+type Types = v.InferOutput<typeof TypesSchema>;
```

Comment:
if it's an object, it's a single thing. Shouldn't have `s` in its variable name

```suggestion
const TypeSchema = v.object({
  name: v.pipe(v.string(), v.minLength(1)),
});
type Type = v.InferOutput<typeof TypeSchema>;
```